doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -218,7 +218,10 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+* :ref:`matter_lock_sample` sample:
+
+  * Fixed an issue that prevented nRF Toolbox for iOS in version 5.0.9
+    from controlling the sample using NUS (Nordic UART Service).
```

Comment:
```suggestion
  * Fixed an issue that prevented nRF Toolbox for iOS in version 5.0.9 from controlling the sample using :ref:`nus_service_readme`.
```
One line, and might as well link to NUS when we mention it here.

Reply:
Done.